Assets/AnkrSDK/Plugins/WalletConnectSharp.Unity/WalletConnectProvider.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using AnkrSDK.WalletConnectSharp.Unity;
+using Object = UnityEngine.Object;
+
+namespace AnkrSDK.Utils
+{
+	public static class WalletConnectProvider
```

Comment:
Let's not put it inside WalletConnectSharp.Unity. I prefer to put it in our SDK folders (well they are all our sdk folders, but it will be easier to distinguish).

Reply:
I had to move it here because otherwise WebGL has compilation problems. WalletConnectSharp.Unity is excluded for compilation for WebGL and if WalletConnectProvider will be inside of AnkrSDK assemblies then it will reference WalletConnectUnityMonoAdapter and WalletConnect from the WalletConnectSharp.Unity assembly and WebGL will not compile. But of course we can use conditional compilation for WalletConnectProvider since its usages in AnkrSDK assemblies are compiled conditionally for non-WebGL platforms. Shall we do it?